WindowsServerDocs/administration/Linux-Package-Repository-for-Microsoft-Software.md

```diff
@@ -25,7 +25,13 @@ Microsoft's Linux Software Repository is comprised of multiple sub-repositories:
 
 ## Configuring the repositories
 
-Repositories can be configured automatically by installing the Linux package that applies to your Linux distribution and version. The package will install the repository configuration along with the GPG public key used by tools such as apt/yum/zypper to validate the signed packages and/or repository metadata.
+Repositories can be configured automatically by installing the Linux package that applies to your Linux distribution and version. The package will install the repository configuration along with the GPG public key used by tools such as apt, yum or zypper to validate the signed packages and/or repository metadata.
```

Comment:
Cosmetic/codestyle: English tends to use the so-called "Oxford comma" when naming several items like this, just to make sure that the last 2 items are not seen as a combination of some sort.
```suggestion
Repositories can be configured automatically by installing the Linux package that applies to your Linux distribution and version. The package will install the repository configuration, along with the GPG public key used by tools such as apt, yum, or zypper to validate the signed packages and/or repository metadata.
```
Also added a comma to make sure that the last part of the sentence is considered as important as the first part.